include/envoy/http/access_log.h

```diff
@@ -114,6 +114,16 @@ class RequestInfo {
    * Set whether the request is a health check request or not.
    */
   virtual void healthCheck(bool is_hc) PURE;
+
+  /**
+   * Set the downstream address
+   */
+  virtual void setDownstreamAddress(std::string address) PURE;
```

Comment:
You mean to say add something like follows 
request_info_.setDownstreamAddress(downstream_address_) in conn_manager_impl.cc.decodeHeaders [here](https://github.com/lyft/envoy/blob/master/source/common/http/conn_manager_impl.cc#L484)